addons/gamification/data/gamification_data.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <!--Ranks-->
+          <record id="rank_newbie" model="gamification.karma.rank">
+              <field name="name">Newbie</field>
+              <field name="description">You just began the adventure! Welcome!</field>
+              <field name="minimum_value">0</field>
```

Comment:
I would set minimum value to 1 in order to avoid updating a lot of unnecessary users.